### PR TITLE
Feature/api implementation

### PR DESCRIPTION
**Pull request by socunanena**

### CODE REVIEW

File: api/src/db/models.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      required: true,
+    },
+    videoUrl: {
+      type: String,
+      required: true,
+    },
+    rating: mongoose.Schema.Types.Decimal128,
+    type: {
+      type: String,
+      enum: ['FT', 'RFT', 'AMRAP', 'CHIPPER', 'STATIONS'],
```
Comment (pachoning) [inline api/src/db/models.js:16]: Could you add EMOM, please?

Comment (pachoning) [inline api/src/db/models.js:16]: Could you add the value of EMOM, please?

File: api/src/repositories/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module.exports = [
+  {
+    name: 'Repositories init',
```
Comment (pachoning) [inline api/src/repositories/index.js:3]: Is this the entrypoint?

**Review by pachoning — CHANGES_REQUESTED** [review]
These are my comments

File: api/src/db/models.js (hunk 1 above)
Comment (socunanena) [reply api/src/db/models.js:16]: Yep, I missed it!

File: api/src/repositories/index.js (hunk 2 above)
Comment (socunanena) [reply api/src/repositories/index.js:3]: Nope

**Review by pachoning — APPROVED** [review]
Ok

**Pull request merged**